README.md

```diff
@@ -52,10 +52,26 @@ will continue running.
 
 ## Using the LSP (Experimental)
 
-This extension can use [Facebook's starlark project](https://github.com/facebookexperimental/starlark-rust) as a language server.
-
-1. Install the LSP using cargo: `cargo install starlark_bin`
-2. Enable the LSP extension by setting `bazel.lsp.enabled` to `true`.
+This extension can use a LSP for Starlark autocompletion.
+
+There are currently three different compatible language servers:
+
+- [Facebook's starlark project](https://github.com/facebookexperimental/starlark-rust)
```

Comment:
In my PR I omitted this one since bazel-lsp below is a fork of the bazel parts of this with extra features and some bugs fixed, and they're planning on removing it soon. Do you think this is reasonable?

Reply:
Yes, I think removing it is reasonable.

I just didn't do so in my pull request here, because I wanted to keep it as little controversial as possible, and I saw removing the Facebook LSP as a potential contention point